libvast/src/system/accountant.cpp

```diff
@@ -314,12 +314,16 @@ accountant(accountant_actor::stateful_pointer<accountant_state> self,
       for (const auto& [key, value] : r) {
         self->state->record(key + ".events", value.events, ts);
         self->state->record(key + ".duration", value.duration, ts);
-        auto rate = value.rate_per_sec();
-        if (std::isfinite(rate))
-          self->state->record(key + ".rate", rate, ts);
-        else
-          self->state->record(key + ".rate",
-                              std::numeric_limits<decltype(rate)>::max(), ts);
+        if (value.events == 0) {
```

Comment:
As discussed on slack, this special case shouldnt be necessary but empirically we observed that we sometimes receive records with 0 events and 0 duration.

(just leaving this comment in case anyone wonders and looks at the PR in the future)